nemo_rl/models/policy/dtensor_policy_worker.py

```diff
@@ -198,10 +203,34 @@ def __init__(
             else None,
         )
 
+        if "reward_model_type" in self.cfg:
```

Comment:
``I'd like to avoid existence checks as things that configure behavior in general. Can we instead follow the pattern of 
thing:
  enabled: true/false?
 
maybe 
`if self.cfg["reward_model_cfg"]["enabled"]:`

then update all configs to include 
```
policy:
  reward_modeling_cfg:
    enable_rm: false
    ...other cfg items
```

Reply:
Please check 9acf2c5a and let me know if it looks good to you (also cc @terrykong for visibility)

I'd rather not update all configs since this is very RM-specific and it would clutter the other configs for no good reason (nobody should set `enabled: true` in other configs).